test/test_sketch.py

```diff
@@ -0,0 +1,58 @@
+# This file is part of arduino-cli.
+#
+# Copyright 2019 ARDUINO SA (http://www.arduino.cc/)
+#
+# This software is released under the GNU General Public License version 3,
+# which covers the main part of arduino-cli.
+# The terms of this license can be found at:
+# https://www.gnu.org/licenses/gpl-3.0.en.html
+#
+# You can be released from the requirements of the above licenses by purchasing
+# a commercial license. Buying such a license is mandatory if you want to modify or
+# otherwise use the software for commercial activities involving the Arduino
+# software without disclosing the source code of your own applications. To purchase
+# a commercial license, send an email to license@arduino.cc.
+import os
+import platform
+
+import pytest
+
+from test.common import running_on_ci
+
+
+@pytest.mark.skipif(running_on_ci() and platform.system() == "Windows",
```

Comment:
The e2e testing in the Github CI windows VM is temporarly disabled for the `sketch new` feature.
We are experiencing an odd behavior for windows VM in the `test` pipeline that we are not able to reproduce on a physical windows machine. The issue seems to be related to the managing of pytest temporary directories.
The test will remain disabled until a root cause is found and the fix introduced as a separate PR.